channels.yaml

```diff
@@ -227,6 +227,13 @@ releases:
     serverArgs: *serverArgs-v3
     agentArgs: *agentArgs-v2
     featureVersions: *featureVersions-v1
+  # v1.22.14+k3s1 was never released
```

Comment:
Does this need to remain?

Reply:
it gives the reader continuity, that way we can quickly and easily pick out missing/skipped versions
it would definitely make me feel better if we left it and it has no effect on functionality either way, plus it matches the format already established in the file

Reply:
I'm curious what the team thinks about this as it'd be a new standard going forward. We have generally not done this and will lead this file to eventually have quite a few of these entries. And this is more for Rancher consumption than anything. I wonder if it's best put somewhere else like a wiki page in the k3s repo or a long lived discussion.

Reply:
ok, I don't mind removing them if you would like to talk about them to a wider audience. Would you like me to remove these other instances of similar comments (not related to this commit)?

1. https://github.com/rancher/kontainer-driver-metadata/blob/fd2ea6e905c82ab2ece78f33600d5f041068636f/channels.yaml#L165
3. https://github.com/rancher/kontainer-driver-metadata/blob/fd2ea6e905c82ab2ece78f33600d5f041068636f/channels.yaml#L203
4. https://github.com/rancher/kontainer-driver-metadata/blob/fd2ea6e905c82ab2ece78f33600d5f041068636f/channels.yaml#L217
5. https://github.com/rancher/kontainer-driver-metadata/blob/fd2ea6e905c82ab2ece78f33600d5f041068636f/channels.yaml#L242
6. https://github.com/rancher/kontainer-driver-metadata/blob/fd2ea6e905c82ab2ece78f33600d5f041068636f/channels.yaml#L253
7. https://github.com/rancher/kontainer-driver-metadata/blob/fd2ea6e905c82ab2ece78f33600d5f041068636f/channels.yaml#L292
8.